waku/v2/protocol/waku_noise/noise.nim

```diff
@@ -7,16 +7,14 @@
 
 {.push raises: [Defect].}
 
-import std/[oids, options]
+import std/[options, tables]
 import chronos
 import chronicles
 import bearssl
+import strutils
```

Comment:
Should go under `std` I think.

Reply:
Addressed in https://github.com/status-im/nim-waku/commit/76494c2a77672c329567f32b5e18bf15319e9b25